Exercises/operators-and-type-conversions/operators-and-type-conversions.js

```diff
@@ -0,0 +1,16 @@
+// Task #1
+a = 10;
+console.log(parseInt(a + '0'));
+
+// Task #2
+const stringValue = `Amazon annual revenue of US$232.887 billion`;
+let r = new RegExp(/\d+.\d+/);
+const annualRevenue = +stringValue.match(r) * 1e9;
+console.log(annualRevenue);
+const monthlyIncome = annualRevenue / 12;
```

Comment:
Here you actually cast Array to Number in interesting way (Because `annualRevenue` is an array). Luckily this works because when you cast array to number it firstly casts to string and then to number, and if your array have only one element then when you convert it to string it could be also converted to number sometimes:
```js
[1, 2, 3].toString(); // '1,2,3'
[1].toString(); // '1' 

// So this works:
[12] * [3] // will convert [12] to '12' and then convert '12' to 12
// 36

// This also works:
[[[12]]] * [[3]] // 36

//But:
[1,2] * 3 // NaN
```
But you should avoid of using luck in your code. If your array had more than one value such trick would not work.